composer.json

```diff
@@ -76,16 +77,25 @@
             "assets:install %PUBLIC_DIR%": "symfony-cmd"
         },
         "post-install-cmd": [
-            "@auto-scripts"
+            "@auto-scripts",
+            "cghooks add --ignore-lock"
         ],
         "post-update-cmd": [
-            "@auto-scripts"
+            "@auto-scripts",
+            "cghooks update"
         ]
     },
     "conflict": {
         "symfony/symfony": "*"
     },
     "extra": {
+        "hooks": {
+            "pre-commit": [
+                "./bin/phpunit",
+                "./vendor/bin/php-cs-fixer fix -v",
+                "./vendor/bin/phpstan analyse"
+            ]
```

Comment:
what do you think about adding
```
"pre-push": [
    "vendor/bin/php-cs-fixer fix -v --diff --dry-run"
],
```

Reply:
this will only show the changes before push? but if everything is already fixed at pre-commit it shouldn't show anything, right?